src/installer/lib/patcher.ts

```diff
@@ -95,6 +96,10 @@ export function patchTSModule(tsModule: TSModule, tsPackage: TSPackage) {
   const isTSC = (filename === 'tsc.js');
   const patchSrc = generatePatch(isTSC);
 
+  const ver = tsPackage.version.split('.')
+  const majorVer = +ver[0];
+  const minorVer = +ver[1];
+
   /* Add diagnostic modification support */
   const moduleSrc = patchModuleDiagnostics(tsModule, tsPackage);
```

Comment:
This throws when attempting to patch in v5.
I also don't understand why we seem to patch the module diagnostics twice, once here ...